### DISCUSSION TITLE
Copilot Student account doesn't work

### DISCUSSION THREAD

**Discussion by deezer257** [community/community#120817] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

I have access to GitHub student developer pack. I was approved and my certificate is also valid. Nevertheless, in the beginning, Copilot worked, but now I can't access GitHub Copilot from VSCode as a student. Does anybody have the same problem?
I always get this error message

![image](https://github.com/community/community/assets/104132290/7b794ba6-4526-401b-8779-fbf26eee25d8)

**Comment by davecheney** [comment, marked as answer]
@deezer257 your student-2022 coupon expired a few days ago. If you are still a current student you can reapply for the GitHub student developer pack which will provide a further two years of Copilot access.

**Reply by davecheney** [reply]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
